### ISSUE TITLE
PNG appears broken on a diagram with background image so can't export as PDF

### ISSUE THREAD

**Issue by ane-gabriela** [xwikisas/application-diagram#132]
Preconditions: Have the diagram app installed with trial.

Steps to reproduce:

1. Import the xar https://up1.xwikisas.com/#mEmNXhJsATTi2Iucy7ZQOw
2. Access the Diagram imported
3. Observe the Attachments tab
4. Edit the diagram and make a change
5. Save and View
6. Observe the Attachments tab
7. Click on More Actions > Export as PDF

Expected results: The PNG image is shown/saved properly and the Diagram can be exported as PDF.

Actual results: The PNG appears broken, even with edit/save it doesn't change and the PDF export returns "%PDF-1.4 %����" "Notice
A problem occurred while trying to process your request. Please contact the webmaster if this happens again."

Environment: Windows 10, XWiki 11.10.4 with MySQL 5.7 and Standard, Chrome 81, Firefox 74

![Firefox](https://user-images.githubusercontent.com/40691112/79005348-536d4100-7b5f-11ea-86cf-96f0eed57d23.jpg)
![Chrome - PNGBroken](https://user-images.githubusercontent.com/40691112/79005350-5405d780-7b5f-11ea-9c98-651779e59552.jpg)

![PDFExport](https://user-images.githubusercontent.com/40691112/79005539-c1196d00-7b5f-11ea-92ba-8163c6fa4935.jpg)

XWiki console log: https://up1.xwikisas.com/#SIWoOZA9azf-Xjlct1fizQ

Page in XWiki when exporting as PDF : https://up1.xwikisas.com/#GZDYcC2yuwN1yxUYu1MIXg

NOTE: tested several times:

1. Get the same issue on diagrams that have image background
2. The diagrams that don't have backgrounds are working fine

**Comment by oanalavinia** [comment]
I could reproduce with the imported diagram, but it is repaired after a new Save of the page. It was the same even before editing it?

> Get the same issue on diagrams that have image background

Meaning that you reproduce the same for new created diagrams with background? Or for other diagrams created in a previous version of Diagram Application?
For me I couldn't reproduce for new diagrams with background, neither for diagrams that were created in Diagram Application 12.3 (previous version) and tested after upgrade. 

The issue with the diagram appeared when you opened the diagram after upgrade? Or you performed any additional steps before observing the broken Attachments Tab or the errors? The background image was added using 'Actual size', or with 'Resize' option?

**Comment by ane-gabriela** [comment]
@oanalavinia The 2 Diagrams that had issues were created with prior version of Diagram. I've edited and saved them but nothing changed.

I've created a new Diagram with background image and it is being saved properly.

I saw the broken png after I changed in Object mode from SVG=Yes to No and made changes to the zoom option - I've tried all options from 1 to 5.

The Images were added from the right panel - one with an image from my computer and one with a link to a image. The sizes were the default.
